README.md

```diff
@@ -118,3 +128,24 @@ By default, `rustc` compiles everything in the `examples/` directory in debug mo
 rust-gdb target/debug/examples/$EXAMPLE_NAME
 $ gdb run
 ```
+# Companies Using Tantivy 
```

Comment:
@maximkpa can you add the images in the repository? I don't want to rely on external images.